extensions/campeonato-brasileiro/src/standings.tsx

```diff
@@ -0,0 +1,81 @@
+import { Action, ActionPanel, Icon, Image, List } from "@raycast/api";
+import { useState } from "react";
+import useStandings from "./hooks/useStandings";
+
+export default function GetStandings() {
```

Comment:
```suggestion
export default function Command() {
```
Here is what I meant about the Commands. I'm adding some suggestion here; the name of the file should match the command name in the manifest, which you did perfect.